src/main/resources/scipooldev.properties

```diff
@@ -11,4 +11,5 @@ SESSION_TAG_CLAIMS=sub,userid,team,user_name,company,given_name,family_name
 # 3409013 scipooldev-external
 TEAM_TO_ROLE_ARN_MAP=[{"teamId":"3407239","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}, \
                       {"teamId":"3409012","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}, \
-                      {"teamId":"3409013","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}]
+                      {"teamId":"3409013","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}  \
```

Comment:
~This is confusing. In the commented teams above 3409011 is not mentioned. Why is there a new team?~ Don't you want to replace the association of ServiceCatalogEndusers to 3409013 with ServiceCatalogExternalEndusers instead?